src/components/SheetBuilder/SheetBuilderForm/SheetBuilderForm.tsx

```diff
@@ -0,0 +1,77 @@
+import { TabContext, TabList, TabPanel } from '@material-ui/lab';
+import { Tab } from '@mui/material';
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import {
+  resetFormAlert,
+  selectFormAlert,
+} from '../../../store/slices/sheetBuilder/sheetBuilderSliceForm';
```

Comment:
eu acho que é interessante que seja definido um alias pra pasta `src` e mudar esse caminho pra algo tipo:
```suggestion
} from '@/store/slices/sheetBuilder/sheetBuilderSliceForm';
```
Isso deixa mais legível e evita que você fique sempre tendo que definir sempre vários directory climbing.

Reply:
Isso serve pra todos os outros casos, uma vez que não é só aqui que está dessa forma.

Reply:
agree total, vou puxar isso